app/next-client-app/types/scanreport.ts

```diff
@@ -13,3 +13,20 @@ interface ScanReport {
   previous: string | null;
   results: ScanReportResult[];
 }
+
+interface ScanReportTables {
```

Comment:
At this point - I feel like these pagination types should be just combined. And they take the result type. The result type should then be the name of the model, not +`Result`

Reply:
Thanks, Andy.
The changes based on your comments will definitely be made on the branch Scanreport-fields.